src/loader/aarch64/pe/mod.rs

```diff
@@ -168,6 +180,40 @@ impl KernelLoader for PE {
     }
 }
 
+/// load uefi for aarch64
+/// uefi image for virtual machine on arm64 is not formatted, so
+/// a special loader for it is needed.
+///
+/// # Arguments
+///
+/// * `guest_mem` - Guest memory address where UEFI should be loaded.
+/// * `guest_addr` - The offset in guest_mem from which the UEFI image will be loaded.
+/// * `uefi_image` - The UEFI image
+#[cfg(target_arch = "aarch64")]
+pub fn load_uefi<F, M: GuestMemory>(
```

Comment:
The code of this functions is rather similar to `load_dtb`, can you refactor to reduce some duplicate code?